### PR TITLE
SUP-32090: Update unique kms new item pending moderation event notifi…

### PR DESCRIPTION
**Pull request by Tova-a**
…cation

### CODE REVIEW

File: deployment/updates/scripts/xml/2022_11_20_updateUniqueKmsNewItemPendingModeration2.template.xml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                        <description>Entry creator ID</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>($entry = entryPeer::retrieveByPk($scope->getObject()->getEntryId())) ? $entry->getFromCustomData('creatorPuserId') : ''</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>creator_name</key>
+                        <description>Entry creator name</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>!is_null(entryPeer::retrieveByPk($scope->getObject()->getEntryId())) ? !is_null($kuser = kuserPeer::retrieveByPk(entryPeer::retrieveByPk($scope->getObject()->getEntryId())->getCreatorKuserId())) ? $kuser->getFirstName() . ' ' .  $kuser->getLastName(): '' : ''</code>
```
Comment (inbal-ben-david) [inline deployment/updates/scripts/xml/2022_11_20_updateUniqueKmsNewItemPendingModeration2.template.xml:91]: no need for additional !is_null checks, keep the original template convention

File: deployment/updates/scripts/xml/2022_11_20_updateUniqueKmsNewItemPendingModeration2.template.xml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                        </value>
+                    </item>>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>creator_email</key>
+                        <description>Entry creator email</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>!is_null(entryPeer::retrieveByPk($scope->getObject()->getEntryId())) ? !is_null(kuserPeer::retrieveByPk(entryPeer::retrieveByPk($scope->getObject()->getEntryId())->getCreatorKuserId())) ? kuserPeer::retrieveByPk(entryPeer::retrieveByPk($scope->getObject()->getEntryId())->getCreatorKuserId())->getEmail() : '' : ''</code>
```
Comment (inbal-ben-david) [inline deployment/updates/scripts/xml/2022_11_20_updateUniqueKmsNewItemPendingModeration2.template.xml:98]: same as above

File: deployment/updates/scripts/2022_11_20_deploy_update_email_event_notification_unique_kms_new_item_pending_moderation_2.php [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?php
```
Comment (inbal-ben-david) [inline deployment/updates/scripts/2022_11_20_deploy_update_email_event_notification_unique_kms_new_item_pending_moderation_2.php:1]: add release notes

**Review by inbal-ben-david — APPROVED** [review]

**Review by inbal-ben-david — APPROVED** [review]

File: deployment/updates/scripts/xml/2022_11_20_updateUniqueKmsNewItemPendingModeration2.template.xml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+                        <description>Entry creator ID</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>($entry = entryPeer::retrieveByPk($scope->getObject()->getEntryId())) ? $entry->getFromCustomData('creatorPuserId') : ''</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>creator_name</key>
+                        <description>Entry creator name</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>($entry = entryPeer::retrieveByPk($scope->getObject()->getEntryId())) ? ($kuser = kuserPeer::retrieveByPk($entry->getCreatorKuserId())) ? $kuser->getFirstName() . ' ' .  $kuser->getLastName(): '' : ''</code>
```
Comment (inbal-ben-david) [inline deployment/updates/scripts/xml/2022_11_20_updateUniqueKmsNewItemPendingModeration2.template.xml:91]: would be nicer to have the condition with (), the last part where you have : '' :'' can be a bit confusing

Comment (Tova-a) [reply deployment/updates/scripts/xml/2022_11_20_updateUniqueKmsNewItemPendingModeration2.template.xml]: Fixed.

**Review by inbal-ben-david — APPROVED** [review]

**Pull request merged**